site2/docs/concepts-messaging.md

```diff
@@ -958,3 +974,7 @@ The following is an example of delayed message delivery for a producer in Java:
 producer.newMessage().deliverAfter(3L, TimeUnit.Minute).value("Hello Pulsar!").send();
 ```
 
+![consumer](https://user-images.githubusercontent.com/94193423/192619010-cb1a5772-f0b8-4d40-b9f9-1df1c5d8ce68.svg)
```

Comment:
@DaveDuggins I guess your intention is to commit the image files through this pull request, so the last 4 lines are not required in the content update. You only need to add the SVG files to your branch.

Reply:
Understood.  Thanks Jun!